easybuild/tools/module_generator.py

```diff
@@ -981,7 +1000,7 @@ def update_paths(self, key, paths, prepend=True, allow_abs=False, expand_relpath
         abspaths = []
         for path in paths:
             if os.path.isabs(path) and not allow_abs:
-                raise EasyBuildError("Absolute path %s passed to update_paths which only expects relative paths.",
+                raise EasyBuildError("Absolute path %s passed to _update_paths which only expects relative paths.",
```

Comment:
Might be worth taking some freedom here to not mention the private method but "pretend" it is the public method.    
That would avoid having to change the tests.

```suggestion
                raise EasyBuildError("Absolute path %s passed to update_paths which only expects relative paths.",
```

Reply:
well, i was gonna try to be precise, but I don't care either way. I can fix the tests if someone wants me to.